sdk/containerregistry/Azure.Containers.ContainerRegistry/TROUBLESHOOTING.md

```diff
@@ -0,0 +1,171 @@
+# Troubleshoot Azure Container Registry client library issues
+
+This troubleshooting guide contains instructions to diagnose frequently encountered issues while using the Azure Container Registry client library for .NET.
+
+## Table of contents
+
+* [General troubleshooting](#general-troubleshooting)
+  * [Enable client logging](#enable-client-logging)
+* [Troubleshooting authentication issues](#troubleshooting-authentication-issues)
+  * [AuthenticationFailedException](#authenticationfailedexception)
+  * [CredentialUnavailableException](#credentialunavailableexception)
```

Comment:
Is it just me or is the deep link here not working?

Reply:
All the links working for me - https://github.com/ShivangiReja/azure-sdk-for-net/blob/ACR_AddTSG/sdk/containerregistry/Azure.Containers.ContainerRegistry/TROUBLESHOOTING.md